src/esp/assets/ResourceManager.h

```diff
@@ -74,7 +97,7 @@ class ResourceManager {
   bool createScene(Importer& importer,
                    const AssetInfo& info,
                    const MeshMetaData& metaData,
-                   scene::SceneNode& parent,
+                   scene::SceneNode& sceneNode,
```

Comment:
Why do you need to rename it?

Reply:
Original `parent` is much better. Renamed back.